From 196966b02bf40a971bdf8888a2fa8ba7db0106f1 Mon Sep 17 00:00:00 2001
From: Gleb Popov <6yearold@gmail.com>
Date: Tue, 24 Oct 2023 22:03:20 +0300
Subject: [PATCH] kcms/region_language: Fix check for GLIBC_LOCALE.

The GLIBC_LOCALE_AUTO is created by the #cmakedefine01 directive and thus is
always defined.
---
 config-workspace.h.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config-workspace.h.cmake b/config-workspace.h.cmake
index b10d8bc4ae..5e252be071 100644
--- config-workspace.h.cmake
+++ config-workspace.h.cmake
@@ -18,6 +18,6 @@
 #cmakedefine01 GLIBC_LOCALE_AUTO
 #cmakedefine01 GLIBC_LOCALE_GENERATED
 
-#if defined(GLIBC_LOCALE_AUTO) || defined(GLIBC_LOCALE_GENERATED)
+#if GLIBC_LOCALE_AUTO == 1 || GLIBC_LOCALE_GENERATED == 1
 #define GLIBC_LOCALE
 #endif
-- 
GitLab
